targets/cortex-m7.json

```diff
@@ -5,5 +5,5 @@
 		"--target=armv7em-none-eabi",
 		"-Qunused-arguments"
 	],
-	"gdb": "arm-none-eabi-gdb"
+	"gdb": ["arm-none-eabi-gdb"]
```

Comment:
Strictly speaking this line is not necessary, but it was already present before so it's fine (no change needed). But you can remove it if you want.

Reply:
Removed because it is defined in the inherited cortex-m.json